typescript-api/src/moonbase/interfaces/augment-api-runtime.ts

```diff
@@ -165,55 +165,6 @@ declare module "@polkadot/api-base/types/calls" {
       /** Generic call */
       [key: string]: DecoratedCallBase<ApiType>;
     };
-    /** 0xbd78255d4feeea1f/4 */
```

Comment:
@timbrinded is it ok that this section disappeared?

Reply:
Are we expecting debugRuntimeApi to be removed in 2900?

Reply:
that's not what is being discussed here. 

The reason the interfaces are being removed is because we are missing V5 config upstream in the PolkadotJS typegen package. We need to raise a corresponding PR there to provide the methods.

Reply:
> we are missing V5 config upstream in the PolkadotJS typegen package.

I know, but polakdotJs only needs runtime api metadata for runtime APIs that must be able to be called externally from js/ts code.
What I said, and remains valid, is that it makes no sense to expose DebugRuntimeApi, because it's an internal API runtime that should only be called by the node itself